ReactNativeTraining/src/app/screens/Home/screens/Chats/index.js

```diff
@@ -1,10 +1,15 @@
-import React from 'react';
+import { connect } from 'react-redux';
 
-import data from '../../../../../utils/constants';
-import ContactList from '../../../components/ContactList/index';
+import Chats from './layout';
 
```

Comment:
We need a container components here, all logic of container here, and this container needs to render a layout component. What you're doing is only connect in another file `Chat` component. If you have doubts of this, tell me